packages/ui/Changeable/Changeable.js

```diff
@@ -170,7 +174,7 @@ const Changeable = hoc(defaultConfig, (config, Wrapped) => {
 			forward(change),
 			({[prop]: value}) => {
 				if (!this.state.controlled) {
-					this.setState({value});
+					this.setState(({value: oldValue}) => value !== oldValue ? {value} : null);
```

Comment:
we discussed that briefly and we could change it.  I don't know if it's possible for an event to happen where the value could be changed twice in a row and the 2nd change (would) change it back to the original and it would be prevented unless we used the callback.